test/regress_dns.c

```diff
@@ -848,6 +848,103 @@ dns_retry_disable_when_inactive_test(void *arg)
 	dns_retry_test_impl(arg, EVDNS_BASE_DISABLE_WHEN_INACTIVE);
 }
 
+static void
+dns_probe_settings_test_impl(void *arg, int flags)
+{
+	struct basic_test_data *data = arg;
+	struct event_base *base = data->base;
+	struct evdns_server_port *port = NULL;
+	struct evdns_base *dns = NULL;
+	int drop_count = 1;
+	ev_uint16_t portnum = 0;
+	char buf[64];
+
+	struct generic_dns_callback_result r1, r2;
+	struct timeval tvalBefore, tvalAfter;
+	unsigned long delta;
+
+	port = regress_get_udp_dnsserver(base, &portnum, NULL,
+	    fail_server_cb, &drop_count);
+	tt_assert(port);
+	evutil_snprintf(buf, sizeof(buf), "127.0.0.1:%d", (int)portnum);
+
+	dns = evdns_base_new(base, flags);
+	tt_assert(!evdns_base_nameserver_ip_add(dns, buf));
+	tt_assert(! evdns_base_set_option(dns, "timeout", "0.2"));
```

Comment:
Let's not use deprecated syntax here and below:

```suggestion
	tt_assert(!evdns_base_set_option(dns, "timeout", "0.2"));
```